spring-cloud-gcp-docs/src/main/asciidoc/spring-integration-storage.adoc

```diff
@@ -0,0 +1,89 @@
+=== Channel Adapters for Google Cloud Storage
+
+The channel adapters for Google Cloud Storage allow you to read and write files to Google Cloud
+Storage through `MessageChannels`.
+
+Spring Cloud GCP provides two inbound adapters, `GcsInboundFileSynchronizingMessageSource` and
+`GcsStreamingMessageSource`, and one outbound adapter, `GcsMessageHandler`.
+
+Maven coordinates, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-integration-gcp</artifactId>
```

Comment:
Agreed

Reply:
Need to add `spring-integration-file` dependency since it isn't going to be polled transitively.
And really explain why. 
That's not addresses. 
Thanks to @meltsufin ! 😄

Reply:
Added a short explanation of why the dependency is required.
Let me know if it's enough or if it needs more explanation.

Reply:
Fine with me. Thanks

Reply:
Since we're being exhaustive here, I guess we should also add `google-cloud-storage` to the dependencies.